src/components/VideoDetails/VideoDetails.test.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import VideoDetails from './VideoDetails.component';
+
+const mockedVideo = {
+  id: { kind: 'youtube#video', videoId: 'nmXMgqjQzls' },
+  snippet: {
+    title: 'Video Tour | Welcome to Wizeline Guadalajara',
+    description:
+      'Follow Hector Padilla, Wizeline Director of Engineering, for a lively tour of our office. In 2018, Wizeline opened its stunning new office in Guadalajara, Jalisco, ...',
+    thumbnails: {
+      default: {
+        url: 'https://i.ytimg.com/vi/nmXMgqjQzls/default.jpg',
+        width: 120,
+        height: 90,
+      },
+    },
+  },
+};
+
+const mockedVideos = [
```

Comment:
same suggestion as in the other test, you could even DRY and reuse that extracted mocked data here